### PR TITLE
1st definitions

### PR DESCRIPTION
**Pull request by p1gp1g**

### CODE REVIEW

File: definitions.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+## Message
+
+This is the content that the application server is sending to the end user application.
+
+## Push System
+
+This is the whole system involved in the delivering of the message from the application server to the end user application.
+
+## Application Push Protocol
```
Comment (sparchatus) [inline definitions.md:19]: I am not sure which protocol this is referring to

File: definitions.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## End User Application
+
+This is the application used by the end user to connect to the application server and to interact with it.
+
+## Message
+
+This is the content that the application server is sending to the end user application.
+
+## Push System
```
Comment (sparchatus) [inline definitions.md:15]: maybe we could more clearly say that this is not the only system but the one that is used when the application would like to receive a message even when not currently running

**Review by sparchatus — COMMENTED** [review]
Very good to lay the foundation using name definitions for the components

File: definitions.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+## Application Server
+
+This is the server that hosts the application.
+
+## End User Application
+
+This is the application used by the end user to connect to the application server and to interact with it.
+
+## Message
```
Comment (p1gp1g) [inline definitions.md:11]: A message can also refer to a message used to communicate with android services. Therefore we'll use Push Message instead.

**Review by sparchatus — APPROVED** [review]

**Review by sparchatus — APPROVED** [review]

**Pull request merged**